_posts/en/newsletters/2021-02-17-newsletter.md

```diff
@@ -144,7 +144,51 @@ software.
 *In this monthly feature, we highlight interesting updates to Bitcoin
 wallets and services.*
 
-FIXME:bitschmidty
+- **Blockstream announces LNsync:**
+  [LNsync][blockstream blog lnsync] allows Lightning wallets to more quickly sync the
+  LN topology in order to optimally route a payment to its destination. An open
+  source plugin, [historian][github historian], provides this functionality for
+  C-Lightning users.
+
+- **Rust light client Nakamoto released:**
+  Alexis Sellier [released Nakamoto][nakamoto blog], a "high-assurance Bitcoin
+  light-client implementation in Rust, with a focus on low resource utilization,
+  modularity and security".
+
+- **Blockstream Satellite broadcasting LN data, Bitcoin Core source:**
+  Blockstream's Satellite, in addition to broadcasting Bitcoin blockchain data,
+  now includes the [Bitcoin Core source code][blockstream satellite bitcoin
+  code], code for a satellite-optimized Bitcoin fork, [Bitcoin
+  Satellite][github bitcoin satellite], and [LN gossip data][blockstream
+  satellite ln data].
+
+- **Blockstream Green implements CSV:**
+  Green Wallet has previously used nLockTime as a wallet recovery mechanism,
+  requiring users to receive backup emails with pre-signed transactions from
+  Blockstream after each transaction in order to recover funds. By [implementing
+  an `OP_CHECKSEQUENCEVERIFY` (CSV) recovery mechanism][blockstream green csv],
+  wallets can be recovered without needing backup files or associating an email
+  address with the wallet.
+
+- **Muun 2.0 released:**
+  The latest [Muun 2.0 release][muun 2.0 blog] includes multisig, wallet recovery
+  features, and also open sources the Android and iOS mobile apps.
+
+- **Joinmarket 0.8.1 released:**
+  The [0.8.1 release][joinmarket 0.8.1] contains additional [PSBT][topic psbt]
+  support for PSBTs created externally, [signet][topic signet] support, a fix
+  for BIP21 (see [Newsletter #127][news127 bech32 casing]). Terminal-based UI
+  for JoinMarket, [JoininBox][github joininbox], was also updated to support 0.8.1.
+
+- **VBTC adds batched segwit withdrawals:**
+  VBTC, a Vietnamese exchange, [added batched segwit withdrawal][vbtc blog
+  segwit] capabilities after recently also [enabling Lightning withdrawals][vbtc blog lightning].
+
+- **Bitcoin Dev Kit v0.3.0 released:**
+  Rust wallet library Bitcoin Dev Kit announced its [v0.3.0 release][bdk
+  0.3.0 blog] including splitting out the CLI into its own repository. The
+  recent [BDK v0.2.0][bdk 0.2.0 blog] release added Branch and Bound (BnB) coin selection,
+  [descriptor][topic descriptors] templates, `sortedmulti`, and more.
```

Comment:
Since `sortedmulti` is part of descriptors, maybe make it a parenthentical: "[...], descriptor templates (including with the recently-added `sortedmulti`), and more"